templates/Image_Stitching.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
<p>If we don't have actual data in the raw folders yet, we should change them to <code>path_to_raw_channel_data</code> or something equivalent so it's clear that they're examples of how the data is formatted. If/when we decide to update the example_dataset to be compatible, we can modify the folders to point to the correct raw data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/702/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>